libmamba/src/core/query.cpp

```diff
@@ -196,12 +234,41 @@ namespace mamba
                 }
             }
 
-            if (!pkg.constrains.empty())
+            if (!buildsByVersion.empty())
             {
-                fmt::print(out, "\n Run Constraints:\n");
-                for (auto& c : pkg.constrains)
+                fmt::print(out, "\n Other Versions ({}):\n\n", buildsByVersion.size());
+
+                std::stringstream buffer;
+
+                using namespace printers;
+                Table printer({ "Version", "Build", "", "" });
```

Comment:
Should this be using `printers::alignmentMarkers.at(printers::alignment::left)` etc?

Reply:
No, as the table is manually created here and not passed to the `table` helper function which contains the logic for detecting the markers.